cdap-kubernetes/src/main/java/io/cdap/cdap/k8s/runtime/KubeTwillController.java

```diff
@@ -343,6 +348,14 @@ public void setJobStatus(V1JobStatus jobStatus) {
     this.jobStatus = jobStatus;
   }
 
+  /**
+   * Sets job timeout status
+   * @param jobTimedOut true if the job timed out, false otherwise
+   */
+  public void setJobTimedout(boolean jobTimedOut) {
```

Comment:
It's better to have a `terminateOnTimeout()` method instead of a setter like this.

Reply:
fixed